query/src/main/scala/hmda/query/model/filing/ModifiedLoanApplicationRegister.scala

```diff
@@ -1,44 +1,43 @@
 package hmda.query.model.filing
 
 case class ModifiedLoanApplicationRegister(
```

Comment:
In addition to this `ModifiedLoanApplicationRegister` class, there's also one defined in `api-model/src/main/scala/hmda/api/model/ModifiedLoanApplicationRegister.scala`. The one in the `api-model` project can be deleted, right?